SWIPs/swip-20.md

```diff
@@ -17,7 +17,7 @@ Currently, a storer node's stake to buy their way into the redistribution game,
 
 ## Objectives
 
-Should the effective price of a user's stake increase, they should be able to withdraw the appreciation, i.e., the portion of the value that is above the amount required to play and safeguard the system.
+Should the effective price of a user's stake dencrease, they should be able to withdraw the appreciation, i.e., the portion of the value that is above the amount required to play and safeguard the system.
```

Comment:
no. if the value increases:

Should the valued of a user's stake increase, they should be able to withdraw the appreciation

Reply:
I already asked about this above, can we check the math as it clearly says

**surplusStake = potentialStakeBalance - effectiveStake**

which means effective stake needs to get lower to be able to take funds out. Both Nugaon and me concluded that  effective stake needs to be lower. If we are not talking here about effective Stake but just price of committed stake in BZZ again it needs to get lower as we are substracting from min(pot,com).
I do wonder if something is wrong here with math or wording . @nugaon  please also jump in here

Reply:
agree as we discussed.
since then, I see the misleading `effective price` has been changed to `value` which is correct.